services/supervisord/devcontainer_test.go

```diff
@@ -221,10 +224,11 @@ func TestDevContainer(t *testing.T) {
 	})
 }
 
-func normalizeFullversion(info *version.PackageInfo) string {
+func normalizeFullversion(info *version.PackageInfo) (version string, isFeatureBranch bool) {
 	fullVersion := info.FullVersion
-	if os.Getenv("FEATURE_BRANCH") != "" {
+	isFeatureBranch = os.Getenv("FEATURE_BRANCH") != "" || strings.HasPrefix(fullVersion, "1:")
```

Comment:
We don't set `FEATURE_BRANCH` anywhere now, I will remove it

Reply:
Cool.